pom.xml

```diff
@@ -76,6 +131,14 @@
 	</modules>
 
 	<properties>
+		<spring-cloud-stream.version>1.0.2.RELEASE</spring-cloud-stream.version>
+		<spring-cloud-deployer-spi.version>1.0.2.RELEASE</spring-cloud-deployer-spi.version>
+		<spring-cloud-deployer-local.version>1.0.2.RELEASE</spring-cloud-deployer-local.version>
+		<spring-cloud-stream-binder-rabbit.version>1.0.2.RELEASE</spring-cloud-stream-binder-rabbit.version>
+		<spring-cloud-deployer-resource-support.version>1.0.2.RELEASE</spring-cloud-deployer-resource-support.version>
+		<spring-cloud-deployer-resource-maven.version>1.0.2.RELEASE</spring-cloud-deployer-resource-maven.version>
```

Comment:
Is this being used?  I see that it is commented out in the spring-cloud-task-core/pom.xml